### ISSUE TITLE
Bug in plot_utils.py

### ISSUE THREAD

**Issue by zhefan** [google-research/rliable#11]
Hi,

In `plot_utils.py`, I think [this line](https://github.com/google-research/rliable/blob/72fc16c31c4021b72e7b21f3ba915e1b38cff481/rliable/plot_utils.py#L245) ought to be `algorithms = list(point_estimates.keys())`
https://github.com/google-research/rliable/blob/72fc16c31c4021b72e7b21f3ba915e1b38cff481/rliable/plot_utils.py#L245
Otherwise, `algorithms` cannot be indexed in the next line.

**Comment by agarwl** [comment]
Yes, it seems that the other plotting functions use `list(point_estimates.keys())` but missed it here. Thanks for catching this .. there are tests for main library functions but not for the plotting functions. Will fix.
